great_expectations/expectations/metrics/column_aggregate_metrics/column_mean.py

```diff
@@ -25,7 +25,7 @@ def _pandas(cls, column, **kwargs):
     def _sqlalchemy(cls, column, **kwargs):
         """SqlAlchemy Mean Implementation"""
         # column * 1.0 needed for correct calculation of avg in MSSQL
-        return sa.func.avg(column * 1.0)
+        return sa.func.avg(1.0 * column)
```

Comment:
Just curious: does this make a difference in the results of the calculation?

Reply:
@Shinnnyshinshin No -- only sense of aesthetics on my part.  Thanks!